app/src/main/java/com/babylon/wallet/android/domain/usecases/assets/GetAccountFirstTransactionDateUseCase.kt

```diff
@@ -0,0 +1,26 @@
+package com.babylon.wallet.android.domain.usecases.assets
+
+import com.babylon.wallet.android.data.repository.cache.database.StateDao
+import com.babylon.wallet.android.data.repository.stream.StreamRepository
+import com.babylon.wallet.android.di.coroutines.DefaultDispatcher
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.withContext
+import javax.inject.Inject
+
+class GetAccountFirstTransactionDateUseCase @Inject constructor(
```

Comment:
This use case seems to not return anything, so maybe rename it to `UpdateAccountFirstTransactionDateUseCase`